ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -296,7 +296,25 @@ export function VariableEditForm(props: VariableEditFormProps) {
                   }
                   label="Allow All option"
                 />
-                <Typography variant="caption">Enables an option to include all variable values</Typography>
+                <Typography mb={1} variant="caption">
+                  Enables an option to include all variable values
+                </Typography>
+                {state.listVariableFields.allowAll && (
+                  <TextField
+                    label="Custom All Value"
+                    value={state.listVariableFields.customAllValue}
+                    onChange={(e) => {
+                      setState((draft) => {
+                        if (e.target.value) {
+                          draft.listVariableFields.customAllValue = e.target.value;
+                        } else {
+                          draft.listVariableFields.customAllValue = undefined;
+                        }
+                      });
+                    }}
+                    helperText="When All is selected, this value will be used"
```

Comment:
I could see us adjusting this help text based on feedback later, I’m sure Garrett will have some ideas (before I watched your Loom I didn’t quite understand what this meant)

Reply:
Yeah agreed